numpy/core/tests/test_fromnumeric.py

```diff
@@ -1,17 +0,0 @@
-from __future__ import division, absolute_import, print_function
```

Comment:
Why remove this file? `fromnumeric.py` is a package containing a number of functions. `put` is also a function implemented in `fromnumeric.py`.